pubspec.yaml

```diff
@@ -51,9 +51,14 @@ dependencies:
   flutter_rating_bar: ^4.0.1
 
   # Internationalization
-  intl: ^0.18.0
+  intl: ^0.17.0
```

Comment:
Had to take this back because it does not seem to be compatible with flutter_localizations